packages/core/CHANGELOG.md

```diff
@@ -7,661 +7,352 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 **Note:** Version bump only for package @featurevisor/core
 
-
```

Comment:
this file is auto-generated by Lerna, so I dunno if we can maintain our preferred Prettier-based formatting for too long 🤔